src/main/java/com/softserveinc/ita/utils/runners/TestRunner.java

```diff
@@ -1,16 +1,15 @@
 package com.softserveinc.ita.utils.runners;
 
-import com.codeborne.selenide.Configuration;
 import org.testng.annotations.BeforeMethod;
 
+import static com.codeborne.selenide.Configuration.*;
 import static com.codeborne.selenide.Selenide.open;
 
 public abstract class TestRunner {
 
     @BeforeMethod
     public void openHomePage() {
-        Configuration.browserSize = "1920x1080";
-        Configuration.timeout = 8000;
-        open("https://rozetka.com.ua/ua/");
+        timeout = 8000;
```

Comment:
config browser size, it always starts windowed

Reply:
my screen resolution is 1366x768, its default value of selenide